module/zfs/dbuf.c

```diff
@@ -807,20 +808,14 @@ dbuf_init(void)
 
 	/*
 	 * Setup the parameters for the dbuf caches. We set the sizes of the
-	 * dbuf cache and the metadata cache to 1/32nd and 1/16th (default)
-	 * of the target size of the ARC. If the values has been specified as
-	 * a module option and they're not greater than the target size of the
-	 * ARC, then we honor that value.
+	 * dbuf metadata cache to 1/16th (default) of the max size of the ARC.
```

Comment:
[nit]  It's actually 1/64th (`dbuf_metadata_cache_shift = 6`).  I have a fix for this in another branch that I'll get opened soon, but if you have a mind to fix it now that would be fine too.

Reply:
Fixed this in the manual. The comment here was removed but I added comments to describe the dbuf cache targets elsewhere, without this error.